tests/functional/centos/7/shrink_osd/group_vars/all

```diff
@@ -1,9 +1,11 @@
 ---
 ceph_origin: repository
 ceph_repository: community
+cluster: ceph
 public_network: "192.168.71.0/24"
 cluster_network: "192.168.72.0/24"
-radosgw_interface: eth1
+monitor_interface: eth1
+copy_admin_key: true
```

Comment:
Why is this needed?

Reply:
@leseb the removal of the `radosgw_interface` resolved an error (shown above in the commit comment). the other two were accidents. i pushed again to remove them.